src/panels/calendar/dialog-calendar-event-editor.ts

```diff
@@ -308,6 +308,10 @@ class DialogCalendarEventEditor extends LitElement {
   }
 
   private async _createEvent() {
+    if (!this._summary || !this._calendarId) {
+      return;
```

Comment:
Let's tell the user why nothing happens.
```suggestion
      this._error = this.hass.localize("...");
      return;
```